addon/components/lt-infinity.js

```diff
@@ -14,13 +14,17 @@ export default Component.extend(InViewportMixin, {
 
   didInsertElement() {
     this._super(...arguments);
+
+    let scrollBuffer = this.get('scrollBuffer');
```

Comment:
Added Ember Suave to fix this kind of issues. It says to use `let` so I'm going with it 😝